### DISCUSSION TITLE
part-ii/2018-132/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#216] [Question comments]
Part II 2018 Representation Theory: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2018-132/

**Comment by Kiolt** [comment]
<details>
<summary>Part (iv).</summary>

The centre is trivial, since otherwise we can quotient by it to get a cyclic group, contradiction G being non-abelian. Then we use Sylow's theorems - there is a normal p-subgroup and p Sylow q-subgroups. 

</details>

**Comment by stevezshen** [comment]
<details>
<summary>Part (iii)</summary>

Use Cauchy's theorem to say there is an element $g$ of order 2. Consider $\langle g\rangle$. It must be normal (why)

</details>
